uPortal-webapp/src/main/webapp/media/skins/common/javascript/uportal/up-autocomplete.js

```diff
@@ -39,7 +39,7 @@ var up = up || {};
             );
 
             var tree = {children: []};
-            $(results).each(function(idx, result) {
+            $(results).each(function (index, result) {
```

Comment:
unicorn aims to have abbreviations like `idx`, `msg`, `e`, spelled out in clear English words.